stories/topics/proc-gcp-deploying-extension-nodes.adoc

```diff
@@ -26,6 +26,11 @@ This field is only used to filter the Network list.
 Do not select the subnet ending with `-aap-proxy-subnet`.
 ====
 +
+[NOTE]
+====
+In case of a error stating `Make sure all fields are correct to continue` and `You must select a Network`, please reselect values from dropdown as mentioned above and click done.
```

Comment:
In case of an error...

Network' . Reselect the values from the menu and click btn:[Done].

Reply:
Updated, thanks